monai/networks/blocks/convolutions.py

```diff
@@ -316,3 +317,40 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         res: torch.Tensor = self.residual(x)  # create the additive residual from x
         cx: torch.Tensor = self.conv(x)  # apply x to sequence of operations
         return cx + res  # add the residual to the output
+
+
+class ClassifierBlock(Convolution):
```

Comment:
As I said earlier, this class probably should go in the same file as the Quicknat definition since it's specific to that network and not really something that, I wouldn't think, would be reused elsewhere.